aptos-move/move-examples/move-tutorial/README.md

```diff
@@ -7,7 +7,7 @@ slug: "move-tutorial"
 
 Welcome to the Aptos Move Tutorial! This is the original Move language and tools tutorial, independent of a particular network, adapted to use Aptos tooling. Here you will learn basic usage of the Move language and tools for compiling, testing, and verifying Move.
 
-This tutorial does not teach you how to use the existing [Aptos Move frameworks](../../../developer-docs-site/docs/guides/move-guides/index.md) or how to run code on the Aptos network. See the [Aptos developer documentation](https://aptos.dev/tutorials/aptos-quickstarts) for those instructions.
+This tutorial does not teach you how to use the existing [Aptos Move frameworks](https://aptos.dev/reference/move) or how to run code on the Aptos network. See the [Aptos developer documentation](../../../developer-docs-site/docs/tutorials/index.md) for those instructions.
```

Comment:
Do we prefer absolute vs relative links? IN the past, Clay was very adamant about all relative

Reply:
I think you need the absolute link to get the aptos.dev style processing, as this is a reference from plain github markdown.

Reply:
+1: The reference page only renders correctly on aptos.dev.